include/uv.h

```diff
@@ -1461,6 +1461,25 @@ struct uv_loop_s {
   /* Internal flag to signal loop stop. */
   unsigned int stop_flag;
   void* reserved[4];
+  /* private fields that appear in both implementations */
+  /* Lists of active loop (prepare / check / idle) watchers */
+  void* prepare_handles[2];
+  void* check_handles[2];
+  void* async_handles[2];
+  /* Timers */
+  void* idle_handles[2];
```

Comment:
Em, do you want me to udpate the comment too then? Is it (async/ prepare/ check/ idle) watchers?